README.md

```diff
@@ -4,44 +4,41 @@
 
 RESTEasy is a JBoss.org project aimed at providing productivity frameworks for developing client and server RESTful applications and services in Java.  It is mainly a JAX-RS implementation but you'll find some other experimental code in the repository.
 
-The project page can be found at https://resteasy.jboss.org
+The project page can be found at [https://resteasy.github.io/](https://resteasy.github.io/).
 
 ## JAX-RS
-RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
 
-Read the entire specification [here](https://github.com/jax-rs)
+RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
 
-The RESTEasy code is in the github repository, (https://github.com/resteasy/Resteasy.git).
+Read the entire specification [here](https://github.com/eclipse-ee4j/jaxrs-api).
```

Comment:
I agree this should be improved and link to Jakarta repo provided. In that case, though, you should also mention it in the text above, maybe replacing the reference to JCP or explaining where the spec is today.

Reply:
okay I'll add the notes in the text